Src/Web/Web.Shared.Net/ExceptionTrackingTelemetryModule.cs

```diff
@@ -67,8 +75,47 @@ public void OnError(HttpContext context)
         /// <param name="configuration">Telemetry configuration to use for initialization.</param>
         public void Initialize(TelemetryConfiguration configuration)
         {
-            this.telemetryClient = new TelemetryClient(configuration);
-            this.telemetryClient.Context.GetInternalContext().SdkVersion = SdkVersionUtils.GetSdkVersion("web:");
+            if (this.EnableMvcAndWebApiExceptionAutoTracking)
+            {
+                if (!this.isInitialized)
+                {
```

Comment:
some other telemetry modules have this logic to ensure they are initialized just once as it seems it's not guaranteed by the base SDK.

Even if the injection is attempted more than once, it could be a bit expensive to run it more than once anyway, so I'd prefer it to be done just once.

Reply:
Sure, but are we forgetting to set isInitialized to true somewhere?

Reply:
nice catch, thanks!